### PR TITLE
:clipboard: Updated README.md

### PR DESCRIPTION
**Pull request by javidiazdom**
## Proposed changes

Cambios en el Readme: explicación del despliegue conjunto con el back-end en un entorno local.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
+Simplemente se ha de ejecutar en este directorio dos instrucciones
+
+```bash
+npm install
+npm run start
+```
+
+Con ello ya estaría disponible la vista en el navegador. :link: http://localhost:3000.
+
+Existen otras uitilidades disponibles para probar y desplegar el código:
```
Comment (zaretmir) [inline frontend/README.md]: Tiny typo in the word "utilidades" 🙈

**Review by zaretmir — COMMENTED** [review]
Hey @javidiazdom ! Although it is not specified in this project, I would rename this file to `README_es.md` to indicate this readme is in Spanish. In the future, we can create a `README.md` (or README_en.md) file with the same content, but in English. It believe it can be a good idea since this is an open source project, and anyone, anywhere in the world, can collaborate.

What do you think?

File: frontend/README.md (hunk 1 above)
Comment (javidiazdom) [reply frontend/README.md:37]: Yeah @zaretmir, that sounds great . Feel free to create said file whenever you like.

**Comment by javidiazdom** [comment]
> Hey @javidiazdom ! Although it is not specified in this project, I would rename this file to `README_es.md` to indicate this readme is in Spanish. In the future, we can create a `README.md` (or README_en.md) file with the same content, but in English. It believe it can be a good idea since this is an open source project, and anyone, anywhere in the world, can collaborate.
> 
> What do you think?

Yeah @zaretmir, that sounds great . Feel free to create said file whenever you like.

**Review by zaretmir — APPROVED** [review]

**Pull request merged**